pkg/universe.dagger.io/alpha/uffizzi/test/test.cue

```diff
@@ -0,0 +1,49 @@
+package uffizzi
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+
+	"universe.dagger.io/alpha/uffizzi"
+)
+
+dagger.#Plan & {
+
+	client: filesystem: "./data": read: {
+		contents: dagger.#FS
+		include: ["*", "./..."]
+	}
+
+	client: commands: sops: {
+		name: "sops"
+		args: ["-d", "../../../../../tests/secrets_sops.yaml"]
```

Comment:
Is it possible to shorter the path by moving `secrets_sops.yaml` to another place?

Reply:
I am gonna loop in @grouville also into this. This secrets file is used by other dagger actions AFAIK. Would have to make changes there also in that case.